recipes/fiji/20141125/meta.yaml

```diff
@@ -0,0 +1,24 @@
+about:
+    home: http://fiji.sc/
+    license: GNU General Public License
+    summary: Fiji is an image processing package—a "batteries-included" distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.
+package:
+    name: fiji
+    version: 20141125
+build:
+  number: 1
```

Comment:
@bgruening Should I just eliminate all OS X components, making this work only for Linux?